### ISSUE TITLE
Map CSV fields

### ISSUE THREAD

**Issue by crisr15** [scientist-softserv/west-virginia-university#7]
# Summary

to be completed after #5 and #26 (does not rely on #27 )

When a user runs an import they need to be able to bring in all metadata fields.  Now that `acda` objects are being created map all metadata fields to bring in all metadata from a properly formatted CSV where the headers match the metadata model.  

Metadata Model: https://docs.google.com/document/d/1zp-amoPPWEM_zMlUxNYGDyvcnGjwGErbHJlHlJV97Ew/edit#heading=h.p2l4eiufm5i1

Metadata Standards: https://docs.google.com/document/d/13NeFGGM4kBUSsebMRDdiviZyoF7Lfc5ivYT4mZDgxBc/edit

# Acceptance Criteria

- [ ] A user can upload a CSV and metadata is available on the work in accordance with the metadata model when they match the metadata standards. 

# Testing Instructions
Client sample Dole CSV for testing can be found: https://drive.google.com/drive/folders/1bHSuGP5Z1RUjo_poMEDXgzBbepEStHXf
(Please add instructions on how to create an importer)

# Notes
This does not rely on `acda_file` being created, just the metadata import.

**Comment by sephirothkod** [comment]
I have been struggling with this, but I think i figured out what is happening. There is stuff coming up on the show page that seems to show that most of the metadata being imported correctly but when I look up the model in the console I see almost every field is blank. 

It looks like everything is being indexed in solr correctly but it is not being saved on the model (fedora). I'm not sure if this is how they want it/how they set it up, so we should look into this more and maybe check with a developer on the client side. Fixing this would likely be out of scope.

fields that are not being saved in either location (fedora/solr) are:
'congress' => { from: ['dcterms:temporal'] },
'physical_location' => { from: ['dcterms:isPartOf'] },
'topic' => { from: ['dcterms:subject'] },
'collection' => { from: ['dcterms:relation'] },
'preview' => { from: ['edm:preview'] },
'publisher' => { from: ['dcterms:publisher'] }

fields being saved only in the model (fedora) are:
'names' => { from: ['dcterms:contributor'] },

**Comment by sephirothkod** [comment]
issues resolved. seemed to be a local issue. 

waiting on client for on collection mappings.

**Comment by DiemBTran** [comment]
**This passes SoftServ QA:** See attached screenshot. Closing because this is not client testable-- there is no staging.

tested on:
- tested locally because staging is not up
- [DoleSampleMetadata v4e.csv](https://github.com/scientist-softserv/west-virginia-university/files/11237282/DoleSampleMetadata.v4e.csv)
- CSV header to property: https://docs.google.com/spreadsheets/d/1_tvpYjDSZaZa7m81mTj6a5TXalRJL601VlwfhJViAMI/edit#gid=0

All CSV headers have been matched to their properties. There are 3 properties that are indeed mapped, but hidden from the work show page (per client set up):
- edm:preview
- edm:isShownAt
- dcterms:isPartOf

<details><summary>All terms are matched</summary>

![DoleSampleMetadata v4e 2023-04-14 at 2 45 18 PM](https://user-images.githubusercontent.com/29311858/232160929-64915c7a-5976-4ec4-9fe9-f28f4a5cccc0.jpg)
</details>
